### PR TITLE
fix: kube to kms auth policy creation

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Addded kube to KMS auth policy creation.

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: dynamic_values/config_modules/service_authorizations/service_authorizations.tf [hunk 1]
```diff
@@ -55,6 +59,22 @@ module "kms_to_block_storage" {
   ]
 }
 
+# workaround for https://github.ibm.com/GoldenEye/issues/issues/6709
```
Comment (ocofaigh) [inline dynamic_values/config_modules/service_authorizations/service_authorizations.tf:62]: use the internal issue https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/issues/645

Comment (Ak-sky) [reply dynamic_values/config_modules/service_authorizations/service_authorizations.tf]: Done.

**Comment by Ak-sky** [comment]
Test is failing with this - 

> Error: UpgradeQuickStartPattern 2024-01-23T11:34:36Z command.go:185: │ Error: [ERROR] Error Deleting resource group: Resource groups with active or pending reclamation instances can't be deleted. Use the CLI commands "ibmcloud resource service-instances --type all" and "ibmcloud resource reclamations" to check for remaining instances, then delete the instances and try again. with response code  {

**Comment by ocofaigh** [comment]
There is a support case open to debug the issue our pipeline is hitting. Seems to be related to backend volume cleanup not being processed by resource controller

**Comment by ocofaigh** [comment]
@Ak-sky hold off on running the pipeline - I bumped the severity of the support case

**Pull request merged**